conf/routes

```diff
@@ -0,0 +1,17 @@
+# Routes
+# This file defines all application routes (Higher priority routes first)
+# ~~~~
+
+# An example controller showing a sample home page
+GET     /                           controllers.HomeController.index
```

Comment:
We'll probably want to get rid of the HomeController eventually, but that can be in a different CL.